packages/react-components/react-persona/docs/Spec.md

```diff
@@ -97,19 +99,23 @@ export type PersonaSlots = {
 
   /**
```

Comment:
You could potentially now just link to the types file so that you don't have to keep updating this section with changes.